doc/microsite/src/main/tut/distage/distage-framework-docker.md

```diff
@@ -19,3 +20,213 @@ libraryDependencies += "io.7mind.izumi" %% "distage-framework-docker" % "$izumi.
 ```
 
 @@@
+
+
+### Overview
+
+Use of `distage-framework-docker` generally follows:
+
+- creating `ContainerDef`s for the containers the application requires
+- declaring a `ModuleDef` that declares the container component
+- including the `DockerSupportModule` in the application's modules
+
+First, the required imports:
+
+```scala mdoc
+import izumi.distage.docker.ContainerDef
+import izumi.distage.docker.Docker.DockerPort
+import izumi.distage.model.definition.ModuleDef
+import izumi.reflect.TagK
+```
+
+#### Create a `ContainerDef`
+
+The `ContainerDef` is a trait that provides a high level interface for defining a docker container resource.
+
+Example nginx container definition:
+
+```scala mdoc
+object NginxDocker extends ContainerDef {
+  val primaryPort: DockerPort = DockerPort.TCP(80)
+
+
+  override def config: Config = {
+    Config(
+      image = "nginx:stable",
+      ports = Seq(primaryPort)
+    )
+  }
+}
+```
+
+#### Declare Container Components
+
+To use this container a module that declares how to make this component is required:
+
+```scala mdoc
+class NginxDockerModule[F[_]: TagK] extends ModuleDef {
+  make[NginxDocker.Container].fromResource {
+    NginxDocker.make[F]
+  }
+}
+```
+
+- `modifyConfig`
+
+- `dependOnDocker` - adds a dependency on a docker container. distage ensures the requested container
+  is available before the dependent is provided.
+
+#### Include `DockerSupportModule`
+
+```scala mdoc:invisible
+
+val _dockerSupportModuleCheck = {
+  import izumi.fundamentals.platform.functional.Identity
+  import izumi.distage.docker.modules.DockerSupportModule
+  (DockerSupportModule, _: DockerSupportModule[Identity])
+}
+
+```
+
+Include the `izumi.distage.docker.modules.DockerSupportModule` module in the application
+modules. This module contains required component declarations and loads the `docker-reference.conf`.
+Container resources depend on these components.
+
+```scala mdoc
+import izumi.distage.docker.modules.DockerSupportModule
+
+object CorrectlyConfiguredApplication {
```

Comment:
If you use objects to prevent print output, I think using `mdoc:silent` might let you avoid that